core/abstract_tests/core/ATS_job-results-success-async-raw-value-multi.adoc

```diff
@@ -1,11 +1,22 @@
-. For each identified process construct an execute request according to test <<ats_core_job-creation-request,/conf/core/job-creation-request>> ensuring that more that one output is requested and the execution mode is set to `async` (i.e. `"mode": "async"`), the requested response type is `raw` (i.e. `"response": "raw"`) and the output transmission is set to `value` (i.e. `"outputTransmission": "value"`) according to requirement <<req_core_job-creation-async-raw-value-multi,/req/core/job-creation-async-raw-value-multi>>.
+. For each identified process construct an execute request according to test <<ats_core_job-creation-request,/conf/core/job-creation-request>> ensuring that asynchronous execution is <<sc_execution_mode,negotiated>> according to test <<ats_core_job-creation-auto-execution-mode,/conf/core/job-creation-auto-execution-mode>>, that the requested response type is `raw` (i.e. `"response": "raw"`) and that the output transmission is set to `value` (i.e. `"outputTransmission": "value"`) according to requirement <<req_core_job-creation-async-raw-value-multi,/req/core/job-creation-async-raw-value-multi>>.
 . Periodically retrieve the status of the asynchronously execute job as per test <<ats_core_job-op,/conf/core/job-op>>.
-. When the job status is `successful`, get the results as per test <<ats_core_job-results-op,/conf/core/job-results>> and verify that they conform to the test <<ats_core_job-creation-async-raw-value-multi,/conf/core/job-creation-async-raw-value-multi>>.
-|===
+. When the job status is `successful`, get the results as per test <<ats_core_job-results-op,/conf/core/job-results>> and verify that they conform to requirement <<req_core_job-results-async-raw-value-multi,/conf/core/job-results-async-raw-value-multi>>.
+|====
+
+NOTE: In the case where a process supports both `async-execute` and `sync-execute` job control options there is a possibility that the server responds synchronously even though the `Prefer` headers asserts a `respond-async` preference.  In this case, the following additional test should be performed.
+
+[width="90%",cols="2,6a"]
+|====
+^|Test Method |. Inspect the headers of the synchronous response and see if a `Link` header is included with `rel=monitor`.
+. If the link exists, get the job status as per test <<ats_core_job-op,/conf/cor e/job-op>> and ensure that the job status is set to `successful`.
+. Get the results as per test <<ats_core_job-results-op,/conf/core/job-results>> and verify that they conform to the test <<ats_core_job-results-async-raw-value-multi,/conf/core/job-results-async-raw-value-multi>>.
+. If the link does not exist then verify that the synchronous response conforms to requirement <<req_core_job-creation-sync-raw-value-multi,/req/core/job-creation-sync-raw-value-multi>>.
+|====
```

Comment:
Maybe we could consider adding the `Preference-Applied` ([RFC-7240](https://datatracker.ietf.org/doc/html/rfc7240#section-3)) header to be more specific about the applied execution mode based on `Prefer`?

Reply:
Great timing!  I was just about to merge.  I'll add a recommendation about Preference-Applied and then merge!